src/discussion-tool-selector/discussion-tool-selector/features-table/FeaturesTable.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+
+export default function FeaturesTable({ forums, featuresList }) {
+  return (
+    <div className="table-responsive features-table border border-info-300 p-3 mb-4">
+      <table className="w-100">
+        <thead>
+          <tr>
+            <th>&nbsp;</th>
+            {forums.map(forum => (
+              <th className="text-center py-3" key={forum.forumId}><h5>{forum.forumId}</h5></th>
```

Comment:
Add some margin around the header so that if there are a couple of long-named forum entries there is some gap between then. 
```suggestion
              <th className="text-center py-3" key={forum.forumId}><h5 className="mx-2">{forum.forumId}</h5></th>
```